README.md

```diff
@@ -1,2 +1,24 @@
-# gcp-deployment-manager
-GCP Deployment Manager scripts to deploy a multi-node YugaByte DB cluster
+# GCP Deployment Manager for YugabyteDB
+
+This repo contains a GCP Deployment manager template to deploy YugaByte DB cluster on GCP. This is an automated deployment that can deploy a multi-region YugabyteDB cluster to GCP. The deployed YugabyteDB cluster gets hosted on 3 nodes residing in 3 separate public subnets and create a universe among them. 
+This repo is ideal to get you running a YugaByte cluster in a few steps. 
+
+# Usage
+
+## Deploying From GCloud 
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FYugaByte%2Fgcp-deployment-manager.git)
```

Comment:
this command would actually clone the repo as well do we still need the step below?